<?xml version="1.0" encoding="UTF-8"?>

<launch>
    <arg name="initial_x" default="-5" />
    <arg name="initial_y" default="0" />
    <arg name="des_x" default="4" />
    <arg name="des_y" default="1" />
    <param name="initial_x" value="$(arg initial_x)" />
    <param name="initial_y" value="$(arg initial_y)" />
    <param name="des_pos_x" value="$(arg des_x)" />
    <param name="des_pos_y" value="$(arg des_y)" />
    <node pkg="spcbot_gazebo" type="follow_wall.py" name="wall_follower" />
    <node pkg="spcbot_gazebo" type="go_to_point.py" name="go_to_point" />
    <node pkg="spcbot_gazebo" type="bug2.py" name="bug2" output="screen" />
</launch>
